openfe/protocols/openmm_utils/system_validation.py

```diff
@@ -93,15 +94,21 @@ def validate_solvent(state: ChemicalSystem, nonbonded_method: str):
       * If there are multiple SolventComponents in the ChemicalSystem.
       * If there is a SolventComponent and the `nonbonded_method` is
         `nocutoff`.
+      * If there is no SolventComponent and no explicitly solvated
+        ProteinMembraneComponent and the `nonbonded_method` is `pme`.
       * If the SolventComponent solvent is not water.
     """
     solv = [comp for comp in state.values() if isinstance(comp, SolventComponent)]
+    # ToDo: Also validate the solvent in the ProteinMembraneComponent?
+    protein_membrane = [
+        comp for comp in state.values() if isinstance(comp, ProteinMembraneComponent)
+    ]
```

Comment:
We could use the new [get_components_of_type](https://github.com/OpenFreeEnergy/gufe/blob/5c281ff4f611a48e2b6791c168bf8fe21431273c/gufe/chemicalsystem.py#L164) method here

Reply:
As the TODO says what happens if the explicit solvent is not water?